### PR TITLE
Use epoch_size instead of hardcoded values

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by szalpal** [comment]
lgtm

File: dali/test/python/test_data_containers.py [hunk 1]
```diff
@@ -77,31 +77,34 @@ def __init__(self, batch_size, num_threads, device_id, num_gpus, data_paths):
...
 LOG_INTERVAL = 200 // BATCH_SIZE + 1
 
 for pipe_name in test_data.keys():
     data_set_len = len(test_data[pipe_name])
     for i, data_set in enumerate(test_data[pipe_name]):
-        iters = data_set[-1]
+        pipes = [pipe_name(batch_size=BATCH_SIZE, num_threads=4, device_id = n, num_gpus = N, data_paths = data_set) for n in range(N)]
+        [pipe.build() for pipe in pipes]
+
+        iters = pipes[0].epoch_size("Reader")
```
Comment (ptrendx) [inline dali/test/python/test_data_containers.py]: Can we here check whether all of the iterators report the same size?

Comment (ptrendx) [reply dali/test/python/test_data_containers.py]: Otherwise lgtm

Comment (JanuszL) [reply dali/test/python/test_data_containers.py]: Done

**Comment by JanuszL** [comment]
Works on 35845.

**Pull request merged**